src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
To get things progressing, I think we'll have to live without Betamax for now. We need https support. Ruby tests using VCR may be implemented/ported using mocks or appropriate fixtures instead.

Also, I think the constants `BASE_URL` and `SEARCH_PATH` might be more suited in the `Client` class. Mostly due to that how we have split in other API clients, but I think it makes sense :)